dashboard/src/app/factories/create-factory/action/factory-action-box.html

```diff
@@ -14,43 +14,46 @@
                    che-place-holder="Enter value"
                    aria-label="Enter value"
                    required
+                   ng-maxlength="255"
                    ng-model="factoryActionBoxCtrl.selectedParam">
           <div ng-message="required">Param is required.</div>
+          <div ng-message="maxlength">The value has to be less than 255 characters long.</div>
```

Comment:
minor issue: is the length is 255 or 256 ? it seems sometimes we have 256 and sometimes 255 (as message is less than, it doesn't include equals)